### DISCUSSION TITLE
what are these checks and why  i have  a failed one

### DISCUSSION THREAD

**Discussion by sahildkun** [Spectrum-CETB/Spectober_Fest#81] [Q&A] [answered]
pls tell me how to fix it ....my code runs perfectly and i file managed just like how other people merged

**Comment by Aditya-150** [comment, marked as answer]
It's bug due to GitHub workflows it will be resolved soon.
Till then have patience.

Thanks ❤️

**Comment by Aditya-150** [comment]
@sahildkun I have mentioned that in the PR. You can just check.
